website/templates/website/components/footer.html

```diff
@@ -1,22 +1,21 @@
 {% load static %}
-<footer class="py-4 bg-dark">
-    <div class="container">
+<footer class="py-4">
+    <div class="container py-4">
         <div class="row justify-content-center">
-            <div class="col-md-4 mr-4">
+            <div class="col-md-4 mr-md-4 mb-4 mb-md-0">
                 <h5>À propos</h5>
-                <div>Lorem ipsum dolor sit amet consectetur adipisicing elit. Neque dolores, veniam sit obcaecati aut odit in assumenda, deleniti, molestias ullam modi.</div>
+                <div class="text-white-50">RedCrarft.org est un serveur Minecraft à but non lucratif ouvert en 2020. Retrouvez plus d'infos sur la page <a href="about.html">à propos & FAQ</a>.</div>
```

Comment:
```suggestion
                <div class="text-white-50">RedCraft.org est un serveur Minecraft à but non lucratif ouvert en 2020. Retrouvez plus d'infos sur la page <a href="about.html">à propos & FAQ</a>.</div>
```

Reply:
Also never link pages directly, I think django has `url()` built in, ask @Likyaz to get absolute links for SEO